lib/glific/providers/maytapi/message.ex

```diff
@@ -61,4 +85,29 @@ defmodule Glific.Providers.Maytapi.Message do
       }
     }
   end
+
+  @spec get_phone_id(map()) :: non_neg_integer()
+  defp get_phone_id(attrs) do
+    WAManagedPhone
+    |> where([g], g.phone == ^attrs.phone)
+    |> select([g], g.phone_id)
+    |> Repo.one!()
+  end
+
+  @spec create_message_after_send(non_neg_integer(), map()) :: any()
+  defp create_message_after_send(org_id, attrs) do
+    message_attrs =
+      %{
+        body: attrs.message,
+        status: "sent",
+        type: "text",
+        receiver_id: Partners.organization_contact_id(org_id),
+        organization_id: org_id,
+        sender_id: Partners.organization_contact_id(org_id),
+        message_type: "WABA+WA",
```

Comment:
Why message_type is "WABA+WA" instead of "WA"?

Reply:
done

Reply:
cool, we will change `sender_id` and `receiver_id`, after the doign this [PR](https://github.com/glific/glific/issues/3377).